src/background/completed-test-step-telemetry-creator.ts

```diff
@@ -40,7 +40,7 @@ export class CompletedTestStepTelemetryCreator {
         this.store.addChangedListener(this.onAssessmentChange);
     }
 
-    private onAssessmentChange = (): void => {
+    private onAssessmentChange = async (): Promise<void> => {
```

Comment:
Why does this method signature need to change? It doesn't look like we call anything async here

Reply:
This is added as a listener on the assessment store, which after this change only accepts async listeners.